src/test/skript/tests/syntaxes/expressions/ExprBlocks.sk

```diff
@@ -0,0 +1,17 @@
+test "blocks void":
+	set {_loc} to location(0.5, 320.5, 0.5)
+	set {_blocks::*} to blocks between {_loc} and ({_loc} ~ vector(10,0,0))
+	assert size of {_blocks::*} is 11 with "Blocks between loc and (loc~vector(10,0,0)) is not 11"
+	assert blocks at {_blocks::*} is void air with "Blocks can be set in the void?"
+	set blocks at {_blocks::*} to stone
+	assert blocks at {_blocks::*} is void air with "Blocks can be set in the void?"
+
+test "blocks vector direction":
+	set {_loc} to (spawn of world "world")
+	set {_blocks::*} to blocks vector(1,0,0) {_loc}
+	assert size of {_blocks::*} is 100 with "Blocks vector(1,0,0) loc is not 100"
+	set blocks at {_blocks::*} to stone
+	assert blocks at {_blocks::*} is stone with "1 or more blocks were not set to stone"
+	set blocks at {_blocks::*} to air
```

Comment:
need to clean up after yourself

Reply:
Same, I dont know what needs to be cleaned up?

Reply:
the blocks were not necessarily air before you set them

Reply:
It doesn't seem like you've actually done what Sovde asked here. You should be returning the blocks to whatever they were before your test ran so that they don't somehow impact a following test (or a subsequent test run).